lib/kafka/concerns/kafka.rb

```diff
@@ -91,7 +91,10 @@ def self.submit_event(current_id:, submission_name:, state:, icn: nil, prior_id:
       'vasi_id' => VASI_ID,
       'system_name' => SYSTEM_NAME,
       'timestamp' => Time.current.iso8601,
-      'additional_ids' => additional_ids
+      'additional_ids' => additional_ids,
+      'context' => {
+        'note' => "Environment: #{Settings.vsp_environment}"
```

Comment:
Rails.env actually doesn't work here. Vets-api has production env for lower environments to my understanding so vsp_environment is the source of truth here.